src/main/java/frc/robot/Robot.java

```diff
@@ -59,6 +60,7 @@ public void robotInit() {
         autoChooser = new SendableChooser<>();
         // autoChooser.setDefaultOption(name, object);
         // autoChooser.addOption(name, object);
+        Logger.configureLoggingNTOnly(this, "Logging");
```

Comment:
`Logger.configureLoggingNTOnly(this, "Logging");` has to be in Robot in order to work since Robot has all of the subsystems.
It doesn't log when you call this function in dashboardDataContainer